pkg/security/http/authz/oauth_scopes.go

```diff
@@ -53,3 +53,23 @@ func findMostRestrictiveAccessLevel(levels []web.AccessLevel) web.AccessLevel {
 	}
 	return min
 }
+
+// Checks whether the user has the requested scope
+func HasScope(user *web.UserContext, scope string) (bool, error) {
+	var claims struct {
+		Scopes []string `json:"scope"`
+	}
+
+	if err := user.Data(&claims); err != nil {
+		return false, fmt.Errorf("could not extract scopes from token: %v", err)
+	}
+	userScopes := claims.Scopes
+
+	for _, userScope := range userScopes {
```

Comment:
Use `slice.StringsContaining()` instead

Reply:
Thanks! I switched to slice.StringsAnyEquals